packages/components/psammead-sitewide-links/src/index.jsx

```diff
@@ -36,6 +37,9 @@ const SitewideLinksWrapper = styled.div`
 const ConstrainedWrapper = styled.div`
   max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
   margin: 0 auto;
+  ${({ isTrustProject }) => {
+    return isTrustProject && `padding-top: ${GEL_SPACING}`;
```

Comment:
Instead of creating a new prop `isTrustProject`, you could do this:

```jsx
${({ trustProjectLink }) => (trustProjectLink && `padding-top: ${GEL_SPACING})`;
```

Reply:
resovled